### PR TITLE
Ask user if they want to download the latest model. #321

### PR DESCRIPTION
**Pull request by ayeankit**
Implementation for `check_new_release` method  and write up code for the user permission if they want to download the latest model.

### CODE REVIEW

**Comment by ayeankit** [comment]
Hello @ethanwhite @henrykironde @bw4sz , I have raise the pull request. Please review it.

**Comment by henrykironde** [comment]
Thanks @ayeankit. 
Before submitting a PR, make sure you have run the tests locally and they have passed. 
Think about a test case for this new function.
Separate the issue-related patch from the clean-up syntax-related patch (mmake another PR for only clean-up).
It appears that you have duplicate code. Try to refactor the code.

**Comment by ayeankit** [comment]
Hey @henrykironde , Do I have to create the test cases?

**"Separate the issue-related patch from the clean-up syntax-related patch (mmake another PR for only clean-up).**"
I didn't get this line. Can you please explain me in more detail?

File: deepforest/utilities.py [hunk 1]
```diff
@@ -63,43 +91,30 @@ def use_bird_release(save_dir=os.path.join(_ROOT, "data/"), prebuilt_model="bird
...
+        
+                print("Downloading model from BirdDetector release {}, see {} for details".format(
+                    _json["tag_name"], _json["html_url"]))
+        
+                with DownloadProgressBar(unit='B',
+                                        unit_scale=True,
+                                        miniters=1,
+                                        desc=url.split('/')[-1]) as t:
+                    urllib.request.urlretrieve(url, filename=output_path, reporthook=t.update_to)
+
```
Comment (henrykironde) [inline deepforest/utilities.py:107]: These are code Clean up changes, make them in a different PR or a different commit.

File: deepforest/utilities.py [hunk 2]
```diff
@@ -47,7 +47,35 @@ def update_to(self, b=1, bsize=1, tsize=None):
             self.total = tsize
         self.update(b * bsize - self.n)
 
-def use_bird_release(save_dir=os.path.join(_ROOT, "data/"), prebuilt_model="bird", check_release=True):
+def check_new_release(save_dir):
```
Comment (henrykironde) [inline deepforest/utilities.py:50]: New code needs a test case.
